lib/plugins/prefixer.js

```diff
@@ -1,15 +1,18 @@
 'use strict';
 
 var htmlAutoprefixer = require('html-autoprefixer');
+var postcssSafeParser = require('postcss-safe-parser');
 
-function PrefixPlugin(options) {
-  this.options = options || {};
+function PrefixPlugin(cheerioOptions, postcssOptions) {
+  this.cheerioOptions = cheerioOptions || {};
+  this.postcssOptions = postcssOptions || {};
+  this.postcssOptions.parser = this.postcssOptions.parser || postcssSafeParser;
```

Comment:
Although on second look, looks like we actually use safe parser already in production mode. https://github.com/Rebelmail/html-uglify/pull/15/files#diff-16b7d75b70953cbf4755508170f2f24cL172